modules/assim.batch/inst/llik.params.csv

```diff
@@ -0,0 +1,7 @@
+"likelihood","parameter","distn","parama","paramb"
+"Gaussian","sd","gamma",0.001,0.001
+"multipGauss","sd","gamma",0.001,0.001
+"multipGauss","bias","normal",0.001,0.001
+"Laplace","alpha1","norm",0.2,0.01
+"Laplace","beta1","norm",-0.2,0.01
```

Comment:
this was my first thought to handle likelihood parameters, but now I don't use this file at all. heteroskedastic Laplacian parameters (`alpha`s `beta`s) are not being sampled, `bias` prior is constructed on the fly, `tau` priors are hardcoded

`tau <- rgamma(1, 0.001 + n[k]/2, 0.001 + error.stats[k]/2)`